integration/test_serveradmin.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+
+
+@pytest.mark.skip("Failing, requires administrator credentials.")
```

Comment:
The reason to skip this, is to highlight the case that in case we ever execute tests for real (not against mock server) then what kind of user credentials should we pass. 
To make sure all the necessary conversations take place.

Reply:
I think this is an on-prem-only endpoint. So you could put `Required on-prem connection` and we can save it for the mock server

Reply:
Changed.